.appveyor.yml

```diff
@@ -20,5 +25,10 @@ build_script:
   - cd build
   - set "PATH=%cd%\bin;%PATH%"
   - call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
-  - cmake -G "NMake Makefiles" -DCMAKE_C_COMPILER=clang-cl -DCMAKE_CXX_COMPILER=clang-cl -DCMAKE_Fortran_COMPILER=flang -DCMAKE_BUILD_TYPE=Release ..
-  - nmake VERBOSE=1
+  - ps: |
+      cmake -G "NMake Makefiles" -DCMAKE_C_COMPILER=clang-cl -DCMAKE_CXX_COMPILER=clang-cl -DCMAKE_Fortran_COMPILER=flang -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=Release ..
+      Push-AppveyorArtifact .\CMakeFiles\CMakeOutput.log
+      Push-AppveyorArtifact .\CMakeFiles\CMakeError.log
+  - ps: |
+      cmake --build . 2>&1 | Out-File build_output.txt
```

Comment:
Doing this will not fail the build

Reply:
That's correct. I thought at the time that the tests would catch it but that's obviously impossible. Sorry for my incompetance.